WNPRC_Purchasing/src/client/components/LineItemRow.tsx

```diff
@@ -59,22 +60,22 @@ export const LineItemRow: FC<LineItemProps> = memo(props => {
                             value={model.item}
                             hasError={model.errors?.find(field => field.fieldName === 'item')}
                             onChange={onValueChange}
-                            isReadOnly={hasRequestId && (isRequester || isReceiver)}
+                            isReadOnly={hasRequestId && ((isRequester && !isReorder) || isReceiver)}
```

Comment:
This somewhat complex expression is repeated a bunch of times. Can we pull it into a single variable and use that each time?